cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamServiceRuntimeModule.java

```diff
@@ -73,6 +74,7 @@ protected void configure() {
         bind(StreamFileJanitorService.class).to(DistributedStreamFileJanitorService.class).in(Scopes.SINGLETON);
         bind(StreamWriterSizeCollector.class).to(BasicStreamWriterSizeCollector.class).in(Scopes.SINGLETON);
         bind(StreamService.class).to(DistributedStreamService.class).in(Scopes.SINGLETON);
+        bind(AbstractNamespaceClient.class).to(DiscoveryNamespaceClient.class).in(Scopes.SINGLETON);
```

Comment:
Yes I think eventually need s ``NamespaceRuntimeModule``.Even that module would need to be added in all 3 modes here, right. Fine for now. Maybe open a TechDebt jira?